QuickFIXn/Logger/LogEventIds.cs

```diff
@@ -0,0 +1,9 @@
+using Microsoft.Extensions.Logging;
+
+namespace QuickFix.Logger;
+
+internal static class LogEventIds
+{
+    internal static readonly EventId IncomingMessage = 7702;
```

Comment:
@jkulubya Is there specific reason you decided on these id values, or was it just an arbitrary choice?

Reply:
@jkulubya Your version had IDs 7702 and 7203 (not 7_7_03), which I'm pretty sure was a typo.  I changed them to 7702/7703.

Let me know if this was your intent.  Also, can you advise how you chose these numbers initially?